timesketch/lib/experimental/win_services.py

```diff
@@ -69,18 +69,22 @@ def win_services(sketch_id):
             sketch_id=sketch_id, query=u'event_identifier:7045'):
         data = event[u'_source'][u'xml_string']
         res = parse_xml(data)
+        res.extend((event.get(u'_index'), event.get(u'_id')))
```

Comment:
This will create multiple edges for one service started multiple times, right?

Reply:
It will in the output of this method, but will be merged when cypher adds them to the graph.

Reply:
Sounds good.